README.adoc

```diff
@@ -18,10 +18,14 @@
 //
 # Microprofile OpenTracing
 
+The Microprofile Opentracing specification defines behaviors and an API for accessing an OpenTracing compliant Tracer object within your JAX-RS application.
+The behaviors specify how incoming and outgoing requests will have OpenTracing Spans automatically created. The API defines how to explicitly access a configured Tracer object.
+
 * Proposal: [MP-0007](0007-DistributedTracing.md)
-* Authors: link:https://github.com/Steve-Fontes[Steve Fontes]
-* Status: **Awaiting review**
-* For the current specification see link:spec/src/main/asciidoc/microprofile-opentracing-spec.asciidoc[microprofile-opentracing-spec].
-* For announcements and more discussions visit link:++https://groups.google.com/forum/#!forum/microprofile++[MicroProfile Google Groups], and filter on link:++https://groups.google.com/forum/#!tags/microprofile/distributed-tracing++[distributed-tracing tag].
+* Authors: link:https://github.com/Steve-Fontes[Steve Fontes],
```

Comment:
Wasn't Kevin a significant author of the OpenTracing effort?  I thought I reviewed a couple of his PRs...